charts/kueue/values.yaml

```
@@ -85,6 +85,7 @@ managerConfig:
     #waitForPodsReady:
     #  enable: false
     #  timeout: 5m
+    #  recoveryTimeout: 3m
```

```suggestion
    #  recoveryTimeout: null # null indicates non-handling of friction between Ready and NotReady
```

IIRC, the default `recoveryTimeout` is null, right?

---

Yes, the default is null if not specified. But to draw attention to this feature we wanted to set 3mins in the manifest - this is commented though, so unless a user uncomment it, it doesn't have affect Kueue's behavior. Discussion:
https://github.com/kubernetes-sigs/kueue/pull/2737#discussion_r1933320560

---

Ah, I remembered that. Thank you!